### DISCUSSION TITLE
Chang style admin dashboard

### DISCUSSION THREAD

**Discussion by Juanes2499** [Budibase/budibase#6995] [Help]
Hello, I Have onw question abuout the style of the tool, is it possibl to chang style of the admin dashboard and give brand style, like icon of my brand and colors of my brand?

**Comment by joebudi** [comment]
Yes. It's possible to customize apps to look fit your brand.

Brief direction:
- [ ] Add your logo to the organization UI in the home portal. 
- [ ] Add your logo to the navigation panel in the design section. 
- [ ] Also in design, theme your app to look on-brand in the theme panel.
